content/en/real_user_monitoring/browser/automatic_setup.md

```diff
@@ -0,0 +1,132 @@
+
+## Prerequisites
+
+The automatic installation method requires that you have the [Datadog Agent][2] installed.
+
+## Setup your RUM application
+
+{{< tabs >}}
+{{% tab "Nginx" %}}
+
+The Datadog RUM Injector leverages the [Nginx Dynamic Modules capability][1] to implement a response body filter. The filter injects the RUM SDK into the response body for responses identified as HTML.
+
+To automatically instrument your RUM application:
+
+1. In Datadog, navigate to the [**Digital Experience > Add an Application Page**][2] and select the JavaScript (JS) application type.
+2. Select **Auto-Instrumentation** and **NGINX**.
+3. Set your Session and Session Replay sample rates. See [guidance on configuring sampling][3].
+4. Copy and run the installer command to load the Datadog RUM SDK Injector onto your Nginx module.
+
+[1]: https://docs.nginx.com/nginx/admin-guide/dynamic-modules/dynamic-modules/
+[2]: https://app.datadoghq.com/rum/list
+[3]: /real_user_monitoring/guide/sampling-browser-plans/
+
+{{% /tab %}}
+{{% tab "Windows IIS" %}}
+
+The Datadog RUM Injector leverages a Windows module that injects the RUM SDK into the response body for responses served by the IIS instance.
+
+1. In Datadog, navigate to the [**Digital Experience > Add an Application Page**][1] and select the JavaScript (JS) application type.
+2. Select **Auto-Instrumentation** and **Windows IIS**.
+3. Set up the IIS module using either the GUI installer or command line as described below:
+
+{{% collapse-content title="Using the GUI installer" level="h4" %}}
+
+1. Download the Datadog RUM installer.
+2. Follow the installer as an administrator by opening the `.msi` file.
+3. Follow the prompts and accept the license agreement.
+4. Set your Session and Session Replay sample rates. See [guidance on configuring sampling][1].
+5. Copy and run the config command shown for each IIS site that you wish to inject RUM into.
+
+[1]: /real_user_monitoring/guide/best-practices-for-rum-sampling/
+
+{{% /collapse-content %}}
+
+{{% collapse-content title="Using the command line" level="h4" %}}
+
+1. Run the Powershell command line as an administrator.
+2. Set your Session and Session Replay sample rates. See [guidance on configuring sampling][1].
+3. Copy and run the config command shown for each IIS site that you wish to inject RUM into.
+
+[1]: /real_user_monitoring/guide/best-practices-for-rum-sampling/
+
+{{% /collapse-content %}}
+
+[1]: https://app.datadoghq.com/rum/list/create/
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Updating your RUM application
+
+{{< tabs >}}
+{{% tab "Nginx" %}}
+
+To update your RUM Application, update your Datadog RUM configs in your `NGINX.conf` file.
+
+{{% /tab %}}
+
+{{% tab "Windows IIS" %}}
+
+To update your RUM application, update the Datadog RUM configs file for the IIS site that you instrumented.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Limitations
```

Comment:
I believe we may be overlooking a significant limitation: currently, it's only possible to configure one RUM Application at a time. What are your thoughts on this, @sarjyusuf?

Reply:
Hey yes, you're right. I mean its not entirely true but maybe we can have another section to talk about multiple app instrumenetation  NIGNX server blocks as advanced readings.